### PR TITLE
JS GROUP CAPSTONE - Show/Movie Webapp

### PR DESCRIPTION
**Pull request by bengigo**
**Features of this project:**

- JavaScript is used to make websites dynamic and build single-page apps.
-  ES6 syntax used.
-  ES6 modules used.
-  Callbacks and promises were used.
-  Webpack used for the whole project
-  JavaScript best practices applied and language style guides in code.
-  Units tests wrote for a JavaScript app.
-  Gitflow followed during the development.
-  Simple git conflicts solved.
-  Data send and receive from an API.
-  Followed API documentation during the whole process.
-  JavaScript code is used asynchronously.

This is a group project completed by [@bengigo](https://github.com/bengigo) [@Therealadetunji](https://github.com/Therealadetunji) and [@29td](https://github.com/29td)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
...
+- Involvement API
+- Jest
+
+## Live Demo
+
+[Live Demo Link](https://drive.google.com/file/d/1GuBA_O6cSGohCIswKyfx4E8-lEnLcqyX/view?usp=sharing)
+
+## Video Presentation
+
+[Video Presentation Link](https://drive.google.com/file/d/1GuBA_O6cSGohCIswKyfx4E8-lEnLcqyX/view?usp=sharing)
```
Comment (Rustamxon7) [inline README.md:23]: - [ ] Good job team 👌🏼👏🏼🙌🏼. As a requirement video presentation should be long max 5 mins. Please rerecord another shorter video 📽. The app is working correctly 😃

File: README.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+
+## 🤝 Contributing
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+-Thank you to Microverse Inc.
+
+## 📝 License
```
Comment (Rustamxon7) [inline README.md:69]: - [ ] [OPTIONAL] Please kindly create an MIT license file and leave a link to it in the readme file 📃. You can see Microverse readme template for this 👍🏼

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @Team, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**